Ad review (#20)
core-wg/core-problem-details: draft-ietf-core-problem-details.md

dereferenced in the normal course of handling problem details (i.e., outside
diagnostic/debugging procedures involving humans).

> An example that contains a custom extension using a URI as `custom-problem-detail-entries`
> key is shown in {{fig-example-custom-with-uri}} (in CBOR diagnostic notation).

Reviewer: the editorial flow is a bit sub-optimal.  What about:

```suggestion
{{fig-example-custom-with-uri}} shows an example (in CBOR diagnostic notation)
of a custom extension using a URI as `custom-problem-detail-entries` key.
```
